bee-network/bee-autopeering/src/init.rs

```diff
@@ -28,11 +28,16 @@ use crate::{
     request::{self, RequestManager, EXPIRED_REQUEST_REMOVAL_INTERVAL},
     server::{server_chan, IncomingPacketSenders, Server, ServerConfig, ServerSocket},
     task::{TaskManager, MAX_SHUTDOWN_PRIORITY},
+    time::SECOND,
 };
 
-use std::{error, future::Future, iter};
+use std::{error, future::Future, iter, time::Duration};
 
 const NUM_TASKS: usize = 9;
+const BOOTSTRAP_MAX_VERIFICATIONS: usize = 10;
+const BOOTSTRAP_VERIFICATION_DELAY: Duration = Duration::from_millis(100);
+const BOOTSTRAP_QUERY_DELAY: Duration = Duration::from_secs(2 * SECOND);
+const BOOTSTRAP_UPDATE_DELAY: Duration = Duration::from_secs(4 * SECOND);
```

Comment:
This looks a bit weird, shouldn't `from_secs` be enough?

Reply:
I know it looks weird in this case. In my own defense: things have changed a bit over time, because the base time unit originally was a millisecond IIRC, so the constant `SECOND` was set to `1000`. Then line 30 would look like:
`const BOOTSTRAP_QUERY_DELAY: Duration = Duration::from_millis(2 * SECOND)`, which I think is a tiny bit more
readable than: 
`const BOOTSTRAP_QUERY_DELAY: Duration = Duration::from_millis(2000)`. Analogously more readable:
`60 * SECOND` rather than `60000` (which actually is an interval autopeering makes use of, and the more readable the bigger intervals you have.
I can also not deny the fact, that this was inspired by the reference Go code I looked at a lot, so it might have had some influence on me :trollface:

Reply:
IMO `Duration::from_millis(2 * SECOND)` is extremely confusing to read

Reply:
And `Duration::from_secs(2 * SECOND)` a bit redundant

Reply:
Agree with @thibault-martinez

Reply:
I find `Duration::from_millis(60000)` not only confusing to read, but also error prone. Now what? :trollface:

Reply:
Why is it error prone ? You ask for milliseconds and then give a raw number. It doesn't get any easier/clearer than that IMO

Reply:
update your browser...you were to quick. I changed my post to make things a bit clearer

Reply:
Yeah yeah I see the point, I didn't initially comment so I'm fine with it anyway, I just find it a bit convoluted for the very simple scenario you are actually having in this file.